src/components/Wiki/Wiki.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import './Wiki.css';
+import backgroundImageStyling from "../../styles/backgroundImageStyling";
+
+import MainNavbar from "../../utils/MainNavbar";
+import EntryMenu from "./EntryMenu";
+import Landing from "./Landing";
+import Entry from "./Entry";
+
+//TODO:
+// [x] remake with bootstrap components
+// [x] use states to switch between Landing/Entry components in the rightPane div
+// [x] flush out edit/comments tab
+// [ ] add crud interactions
+
+//get navbar logo
+const logo = require('../../images/swarmLogoIcon.png');
+const background = backgroundImageStyling();
+
+interface wikiState{
+    view: string;
+}
+
+//component has no props, hence {}
+class Wiki extends React.Component<{}, wikiState>{
+    constructor(props = {}) {
+        super(props);
+        this.rightPaneHandler = this.rightPaneHandler.bind(this);
+        this.state = {
+            view: "landing"
+        }
+    }
+
+    rightPaneHandler(entryId: string){
+        this.setState({
+            view: entryId
+        });
+    }
+
+    componentDidMount() {
+        // @ts-ignore, object could possibly be null
```

Comment:
We want to make sure we are not using TS-Ignore as much as possible. I would implement something that would allow for a case of null.